[CI] XPackRestIT test {p0=ml/3rd_party_deployment/Test start and stop multiple deployments} failing

Apologies if this is just a one-time fluke!

**Build scan:**
https://gradle-enterprise.elastic.co/s/x25bawibf5cuw/tests/:x-pack:plugin:yamlRestTest/org.elasticsearch.xpack.test.rest.XPackRestIT/test%20%7Bp0=ml%2F3rd_party_deployment%2FTest%20start%20and%20stop%20multiple%20deployments%7D

**Reproduction line:**
```
./gradlew ':x-pack:plugin:yamlRestTest' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=ml/3rd_party_deployment/Test start and stop multiple deployments}" -Dtests.seed=FB308B25264ABE4 -Dtests.configure_test_clusters_with_one_processor=true -Dtests.locale=id -Dtests.timezone=Europe/Vienna -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
No

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.test.rest.XPackRestIT&tests.test=test%20%7Bp0%3Dml/3rd_party_deployment/Test%20start%20and%20stop%20multiple%20deployments%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [ml/3rd_party_deployment:442]: expected [2xx] status code but api [ml.start_trained_model_deployment] returned [429 Too Many Requests] [{"error":{"root_cause":[{"type":"status_exception","reason":"Could not start deployment because no ML nodes with sufficient capacity were found","stack_trace":"org.elasticsearch.ElasticsearchStatusException: Could not start deployment because no ML nodes with sufficient capacity were found\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:514)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\t... 6 more\n"}],"type":"status_exception","reason":"Could not start deployment because no ML nodes with sufficient capacity were found","caused_by":{"type":"illegal_state_exception","reason":"Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]","stack_trace":"java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\n"},"stack_trace":"org.elasticsearch.ElasticsearchStatusException: Could not start deployment because no ML nodes with sufficient capacity were found\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:514)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\t... 6 more\n"},"status":429}]

  at __randomizedtesting.SeedInfo.seed([FB308B25264ABE4:87E73768FC98C61C]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:572)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:524)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

  Caused by: java.lang.AssertionError: expected [2xx] status code but api [ml.start_trained_model_deployment] returned [429 Too Many Requests] [{"error":{"root_cause":[{"type":"status_exception","reason":"Could not start deployment because no ML nodes with sufficient capacity were found","stack_trace":"org.elasticsearch.ElasticsearchStatusException: Could not start deployment because no ML nodes with sufficient capacity were found\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:514)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\t... 6 more\n"}],"type":"status_exception","reason":"Could not start deployment because no ML nodes with sufficient capacity were found","caused_by":{"type":"illegal_state_exception","reason":"Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]","stack_trace":"java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\n"},"stack_trace":"org.elasticsearch.ElasticsearchStatusException: Could not start deployment because no ML nodes with sufficient capacity were found\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:514)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.rebalanceAssignments(TrainedModelAssignmentClusterService.java:493)\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.lambda$rebalanceAssignments$8(TrainedModelAssignmentClusterService.java:425)\n\tat org.elasticsearch.server@8.8.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:916)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1623)\nCaused by: java.lang.IllegalStateException: Could not start deployment because no suitable nodes were found, allocation explanation [Could not assign (more) allocations on node [NvbKq8r5RlqHLvo-fZ2i2Q]. Reason: This node has insufficient allocated processors. Available processors [1], free processors [0], processors required for each allocation of this model [1]]\n\tat org.elasticsearch.ml@8.8.0-SNAPSHOT/org.elasticsearch.xpack.ml.inference.assignment.TrainedModelAssignmentClusterService.checkModelIsFullyAllocatedIfScalingIsNotPossible(TrainedModelAssignmentClusterService.java:513)\n\t... 6 more\n"},"status":429}]

    at org.junit.Assert.fail(Assert.java:88)
    at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:373)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:552)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:524)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke(Method.java:578)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
    at java.lang.Thread.run(Thread.java:1623)

```

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/3rd_party_deployment.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/3rd_party_deployment.yml
@@ -423,6 +423,7 @@ setup:
       ml.start_trained_model_deployment:
         model_id: test_model
         deployment_id: test_model_for_search
+        priority: low
         wait_for: started
   - match: {assignment.assignment_state: started}
   - match: {assignment.task_parameters.model_id: test_model}
@@ -443,6 +444,7 @@ setup:
       ml.start_trained_model_deployment:
         model_id: test_model
         deployment_id: test_model_for_ingest
+        priority: low
         wait_for: started
   - match: {assignment.assignment_state: started}
   - match: {assignment.task_parameters.model_id: test_model}

